<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
    <testsuite name="force.apex" timestamp="2025-07-16T05:35:26.000Z" hostname="https://orgfarm-7d18613451-dev-ed.develop.my.salesforce.com" tests="1" failures="1"  errors="0"  time="0.06">
        <properties>
            <property name="outcome" value="Failed"/>
            <property name="testsRan" value="1"/>
            <property name="passing" value="0"/>
            <property name="failing" value="1"/>
            <property name="skipped" value="0"/>
            <property name="passRate" value="0%"/>
            <property name="failRate" value="100%"/>
            <property name="testStartTime" value="Wed Jul 16 2025 12:35:26 AM"/>
            <property name="testSetupTimeInMs" value="0"/>
            <property name="testExecutionTime" value="0.06 s"/>
            <property name="testTotalTime" value="0.06 s"/>
            <property name="commandTime" value="0.61 s"/>
            <property name="hostname" value="https://orgfarm-7d18613451-dev-ed.develop.my.salesforce.com"/>
            <property name="orgId" value="00DgK000001suJEUAY"/>
            <property name="username" value="dany338919@agentforce.com"/>
            <property name="testRunId" value="707gK000008uiLU"/>
            <property name="userId" value="005gK000001VTlBQAW"/>
        </properties>
        <testcase name="testGenerateAditionalInformation" classname="ProductInvocableActionTest" time="0.06">
            <failure message="System.AssertException: Assertion Failed: Debe contener el ID"><![CDATA[Class.ProductInvocableActionTest.testGenerateAditionalInformation: line 18, column 1]]></failure>
        </testcase>
    </testsuite>
</testsuites>
